package.json

```diff
@@ -43,11 +50,16 @@
     "lint-staged": "10.1.2",
     "ora": "4.0.3",
     "prettier": "2.0.4",
-    "deepmerge": "^4.2.2"
+    "react": "^17.0.2"
```

Comment:
React & Jest is not needed at this level, I don't intend to write test cases for generator. This setup was needed or needed to be improved under template/*